src/container/RelayContainer.js

```diff
@@ -445,7 +438,7 @@ function createContainerComponent(
         }
         if (!deferredSubscriptions.hasOwnProperty(subscriptionKey)) {
           deferredSubscriptions[subscriptionKey] =
-            GraphQLDeferredQueryTracker.addListenerForFragment(
+            deferredQueryTracker.addListenerForFragment(
```

Comment:
@josephsavona Is there some other way we can handle deferred queries? It seems kinda painful to add this to the public API for a single thing to support it... that isn't even OSS supported yet :smile: